Files/Filesystem/ListedItem.cs

```diff
@@ -217,9 +217,7 @@ public ListedItem(string folderRelativeId, string dateReturnFormat = null)
         }
 
         public ListedItem()
-        {
-
-        }
+        { }
```

Comment:
```suggestion
        { 
        
        }
```

Reply:
That was to avoid error on GitHub repository code checker